boot/bootutil/src/image_ecdsa.c

```diff
@@ -42,6 +42,7 @@ bootutil_verify_sig(uint8_t *hash, uint32_t hlen, uint8_t *sig, size_t slen,
     int rc;
     bootutil_ecdsa_context ctx;
     FIH_DECLARE(fih_rc, FIH_FAILURE);
+#ifndef MCUBOOT_BUILTIN_KEY
```

Comment:
I would do two implementation of a ~function for call it there, one for MCUBOOT_BUILTIN_KEY and second for  MCUBOOT_BUILTIN_KEY